aiven/provider.go

```diff
@@ -5,14 +5,15 @@ package aiven
 import (
 	"context"
 	"fmt"
-	"github.com/aiven/terraform-provider-aiven/pkg/cache"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"log"
 	"net/url"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/aiven/terraform-provider-aiven/pkg/cache"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
```

Comment:
this whitespace is weird

Reply:
I did `goimports  -w .` and as you can see `golangci-clint -E goimports` is also has no complaints. How different should this be from what we have?

Reply:
i think goimports does not care about grouping of the imports; i think we should group all external imports together ( its only 3 files that need this post processing )